DCScribe.Postgres/Documentation/configuration.Sample.yaml

```
@@ -0,0 +1,85 @@
+---
+# The following setup is for the logging system. For information on how to change this
+# and what other options are available see https://serilog.net/
+Serilog:
+  Using:
+    - Serilog.Sinks.File
+  MinimumLevel: 
+    Default: Information
+  Override: 
+    Microsoft: Warning
+    System: Warning
+  Enrich:
+    - FromLogContext
+    - WithProcessId
+    - WithThreadId
+  WriteTo:
+    - 
+      Name: File
+      Args:
+        path: logs/dcscribe.log
+# This is the configuration for each DCS GameServer. This Sample configuration contains
+# configuration for two separate servers. Edit the configuration below, including adding
+# or removing servers, to suit your setup.
+GameServers:
+  - # These dashes indicate a configuration record for a new GameServer
+    # Replace the following example fields with those used by your server and database
+    Name: RurouniJones Bestest Server
+    # Used in logging
+    ShortName: RBS
+    Database:
+      # Make sure you set your pg_hba.conf file if you are not connecting over localhost
+      # https://www.postgresql.org/docs/current/auth-pg-hba-conf.html
+      Host: rbs.example.com
+      Port: 5432
+      Name: rbs_database
+      Username: dcscribe
+      Password: this_is_a_password
+    # This is the configuration for connecting to the DCS-gRPC server
+    Rpc:
+      # Because DCS-gRPC runs inside the DCS process this should be your DCS server hostname
+      # or IP address
+      Host: rbs.example.com
+      # This is the default port, change it if you have changed the server-side one
+      Port: 50051
+    # Enable/disable the various DCScribe tasks and set task properties e.g. time between queries
+    Tasks:
+      # Stream the units in the mission running on the DCS server
```

```suggestion
      # Record positions of units in the mission running on the DCS server. The Timer value determines how often the unit positions are written to the database. For best accuracy have the PollRate be half of the timer. 
```

I am still not sure if we actually want the PollRate as a config value or if we should mathmatically calculate it to be half of the Timer value. Having it half meants we get two updates per DB write which should make sure all units are cycled through at least once per DB write.